lib/securerandom.rb

```diff
@@ -290,6 +295,8 @@ def uuid
     result
   end
 
+  alias_method :random_string, :choose
+
```

Comment:
```suggestion
```

Reply:
Hi, sorry for the long radio-silence. I got distracted and forgot I was working on this. Here is the PR: https://github.com/ruby/securerandom/pull/14